_data/simple-icons.json

```diff
@@ -2005,6 +2005,11 @@
             "hex": "FF9400",
             "source": "http://design.firefox.com/photon/visuals/product-identity-assets.html"
         },
+        {
+            "title": "Musescore",
```

Comment:
I would say the name of the brand is "MuseScore" (rather then "Musescore"). See e.g. [Wikipedia](https://en.wikipedia.org/wiki/MuseScore) or [a Google Search](https://www.google.com/search?q=musescore).

```suggestion
            "title": "MuseScore",
```

<sup>Note that the the `<title>` in the SVG also has to be updated</sup>

Reply:
Update the title in the JSON and the SVG.